src/prisma/schema.prisma

```diff
@@ -2240,3 +2252,72 @@ model RewardsGithubRepo {
   @@unique([connectionId, repositoryId])
   @@index([connectionId])
 }
+
+model SynapsCredential {
+  id      String  @id @default(uuid()) @db.Uuid
+  apiKey  String
+  secret  String?
+  spaceId String  @unique @db.Uuid
+  space   Space   @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@index([spaceId])
+}
+
+enum SynapsUserKycStatus {
+  APPROVED
+  REJECTED
+  RESUBMISSION_REQUIRED
+  SUBMISSION_REQUIRED
+  PENDING_VERIFICATION
+}
+
+model SynapsUserKyc {
+  id        String               @id @default(uuid()) @db.Uuid
+  deletedAt DateTime?
+  sessionId String // Unique identifier for the user KYC session in Synaps
+  status    SynapsUserKycStatus?
+  userId    String               @unique @db.Uuid
+  user      User                 @relation(fields: [userId], references: [id], onDelete: Cascade)
+  spaceId   String               @db.Uuid
+  space     Space                @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@unique([userId, spaceId])
+  @@index([userId, spaceId])
+}
+
+model PersonaCredential {
+  id         String  @id @default(uuid()) @db.Uuid
+  apiKey     String
+  envId      String
+  templateId String
+  secret     String?
+  spaceId    String  @unique @db.Uuid
+  space      Space   @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@index([spaceId])
+}
+
+enum PersonaUserKycStatus {
+  created
+  pending
+  completed
+  failed
+  expired
+  needs_review
+  approved
+  declined
+}
+
+model PersonaUserKyc {
+  id        String                @id @default(uuid()) @db.Uuid
+  deletedAt DateTime?
```

Comment:
@valentinludu i think we should add createdAt here in case we want to know when they kyc'd. i believe alex/chris mentioned we'd want it to expire

Reply:
Added!

Yes, in the future we can trigger a reevaluation event in Persona after some time passed. e.g. After 1 year or so.
We can discuss more in our meeting what can we do with it.